src/server/utils/response.utils.js

```diff
@@ -1,45 +1,60 @@
-const { resolveFromVersion } = require('./resolve.utils');
+const {resolveFromVersion} = require('./resolve.utils');
 const errors = require('./error.utils');
 
 /**
-* @description When resources are read in the controller functions
-* they all need to respond in a similar manner
-* @function handleSingleReadResponse
-* @param {Express.response} res - Express response object
-* @param {function} next - next function from express middleware
-* @param {string} base - Which spec version is this request coming from
-* @param {T} Resource - Resource class to use for the results
-* @param {object} resource_json - resulting json to be passed in to the class
-*/
-let handleSingleReadResponse = (res, next, base, Resource, resource_json) => {
-	if (resource_json) {
+ * @description When resources are read in the controller functions
+ * they all need to respond in a similar manner
+ * @function handleSingleReadResponse
+ * @param {Express.response} res - Express response object
+ * @param {function} next - next function from express middleware
+ * @param {string} base - Which spec version is this request coming from
+ * @param {T} Resource - Resource class to use for the results
+ * @param {object} resource_json - resulting json to be passed in to the class
+ */
+
+let handleSingleReadResponse = (res, next, base, Resource, resource_json, version_id) => {
+	let {version: resource_version} = resource_json;
```

Comment:
This should go after if (resource_json) {

Reply:
Ok with the move to after "if (resource_json)".
We might need to discuss more regarding how versions show up. I think history version shows up in the profile as followed:
{
	// 	'resourceType': 'Patient',
	// 	'id': '1',
	// 	'meta': {
	//        'lastUpdated' : "2015-09-30T14:31:27.885+00:00",
	// 	   'versionId': '1001'
	// 	},...
}